analyzer/src/funTest/assets/projects/synthetic/poetry-expected-output.yml

```diff
@@ -0,0 +1,772 @@
+          hash:
+            value: "b458f1ceb9371d9d3051de2fd2c45bb8"
+            algorithm: "MD5"
+        vcs:
+          type: ""
+          url: ""
+          revision: ""
+          path: ""
+        vcs_processed:
+          type: "Git"
+          url: "https://github.com/python/tzdata.git"
+          revision: ""
+          path: ""
+      curations: []
+    - package:
+        id: "PyPI::tzlocal:4.2"
+        purl: "pkg:pypi/tzlocal@4.2"
+        authors:
+        - "Lennart Regebro"
+        declared_licenses:
+        - "MIT"
+        - "MIT License"
+        declared_licenses_processed:
+          spdx_expression: "MIT"
+          mapped:
+            MIT License: "MIT"
+        description: "tzinfo object for the local timezone"
+        homepage_url: "UNKNOWN"
+        binary_artifact:
+          url: "https://files.pythonhosted.org/packages/31/b7/3bc2c1868f27677139b772e4fde95265b93151912fd90eb874827943bfcf/tzlocal-4.2-py3-none-any.whl"
+          hash:
+            value: "b29735d51d8961fa3831484ba38a6f7f"
+            algorithm: "MD5"
+        source_artifact:
+          url: "https://files.pythonhosted.org/packages/7d/b9/164d5f510e0547ae92280d0ca4a90407a15625901afbb9f57a19d9acd9eb/tzlocal-4.2.tar.gz"
+          hash:
+            value: "b20f960e77612ff35c5105bb876f0853"
+            algorithm: "MD5"
+        vcs:
+          type: ""
+          url: ""
+          revision: ""
+          path: ""
+        vcs_processed:
+          type: ""
+          url: ""
+          revision: ""
+          path: ""
+      curations: []
+    - package:
+        id: "PyPI::urllib3:1.26.9"
+        purl: "pkg:pypi/urllib3@1.26.9"
+        authors:
+        - "Andrey Petrov"
+        declared_licenses:
+        - "MIT"
+        - "MIT License"
+        declared_licenses_processed:
+          spdx_expression: "MIT"
+          mapped:
+            MIT License: "MIT"
+        description: "HTTP library with thread-safe connection pooling, file post,\
+          \ and more."
+        homepage_url: "https://urllib3.readthedocs.io/"
+        binary_artifact:
+          url: "https://files.pythonhosted.org/packages/ec/03/062e6444ce4baf1eac17a6a0ebfe36bb1ad05e1df0e20b110de59c278498/urllib3-1.26.9-py2.py3-none-any.whl"
+          hash:
+            value: "21785c24a168462568f9a9a0661bcd5a"
+            algorithm: "MD5"
+        source_artifact:
+          url: "https://files.pythonhosted.org/packages/1b/a5/4eab74853625505725cefdf168f48661b2cd04e7843ab836f3f63abf81da/urllib3-1.26.9.tar.gz"
+          hash:
+            value: "d4b58522821a33c5e421191b83e0dbac"
+            algorithm: "MD5"
+        vcs:
+          type: ""
+          url: ""
+          revision: ""
+          path: ""
+        vcs_processed:
+          type: "Git"
+          url: "https://github.com/urllib3/urllib3.git"
+          revision: ""
+          path: ""
+      curations: []
+    dependency_graphs:
```

Comment:
The expected output files should use the dependency tree representation instead of the dependency graph representation, because it is human readable. If you generate the file you can use the function `withResolvedScopes()` on the `AnalyzerResult` class to get the dependency tree representation before serialization. Compare for example with `pip-expected-output.yml`.